📝 Guides (#12)
r-cha/rochadrums: src/content/guides/getting-started-drumming.md

My first kit was an orange Mapex M, these days I think the equivalent is the [Mapex Mars Birch shell pack](https://www.mapexdrums.com/us/products/drums/drum-sets/mars-birch/mars-birch-shell-pack).

Electronic drum sets are great if you live in an apartment or need low-volume, and the entry point is slightly lower.
I have never owned an electronic kit, so I'll point you to [Sweetwater's recommendations here](https://www.sweetwater.com/insync/best-electronic-drum-sets-for-beginners/).

#### Cymbals

Cymbals also have a steep dropoff in quality if you go too low.
The number one thing to look for is material, again: **do not buy brass cymbals**.
They sound awful, break quickly, and feel different to play.
I recommend starting with B8 cymbals; they are bronze like professional cymbals, but with a slightly different alloy composition.
My first cymbals were Zildjian ZBT's, and the modern equivalent would be the Zildjian I family (also check out Sabian B8X or Paiste PST5).

#### Hardware

I encourage you once more to search for quality in your drum hardware.

> Double-braced stands are a **must** for stability while playing and longevity if you have to move them around a lot.

Author: ```suggestion
Double-braced stands are a **must** for stability while playing and durability if you need to move your kit often.
```

There are a lot of options here; look at reviews if you're in doubt or overwhelmed.

With all that acquired, you're ready to jam out!

## Resources worth your time

#### Rudiments

The [Vic Firth Education section](https://ae.vicfirth.com/education/40-essential-rudiments/) has a lot of great content, including the famous rudiments.
Print them out and keep them on hand wherever you practice!

#### Stick Control by George Lawrence Stone

This is the bible of drumming.
You can find it here: [Stick Control by George Lawrence Stone](https://www.stonepercussionbooks.com/stick-control.html).
If you buy only one book in your first year of drumming, let it be this.

> If you don't know how to read music, now's the time to learn!

Author: ```suggestion
If you buy only one book in your first year of drumming, let it be this.

If you don't know how to read music notation, now's the time to learn!
```

> #### Online async lessons

Author: ```suggestion
#### Drum Lessons
```

In-person, one-on-one lessons are the best way to learn **any** instrument or skill.
If you're in Phoenix, AZ, I can be your teacher!
However, there is also a wealth of knowledge from the best drummers of our time available on the internet.

> I love [MikesLessons.com](https://www.mikeslessons.com/) (from Mike Johnston) or [Drumeo](https://www.drumeo.com/); both have free content on YouTube and additional paid content on their sites (Method or Foundations learning tracks are a good place to start).

Author: ```suggestion

There is also a wealth of knowledge from the best drummers of our time available on the internet.
I love [MikesLessons.com](https://www.mikeslessons.com/) (from Mike Johnston) or [Drumeo](https://www.drumeo.com/); both have free content on YouTube and additional paid content on their sites (Method or Foundations learning tracks are a good place to start).
```
